public/src/components/app.jsx

```diff
@@ -7,14 +7,15 @@ import { logoutUser } from '../actions/login';
 
 class App extends Component {
   render() {
-    const { auth, account_info, logoutUser, history } = this.props;
+    const { auth, account_info, history } = this.props;
+    const logoutUserAction = this.props.logoutUser;
```

Comment:
Yes, the linter was unhappy that I was importing logout user above as a const 'logoutUser', then deconstructing it from this.props again, essentially trying to reassign the variable name. With a const declaration this isn't allowed.